utils/test_helper.py

```diff
@@ -0,0 +1,183 @@
+from django.test import Client
+
+import os
+import random
+import string
+import filecmp
+import shutil
+
+from problem.models import Problem, Tag, Testcase, Submission
+from users.models import User
+from utils import config_info
+
+TESTCASE_PATH = config_info.get_config('path', 'testcase_path')
+TEST_PATH = ""
+
+def create_test_directory(dir_name):
+    TEST_PATH = dir_name
+    if not os.path.isdir(TEST_PATH):
+        os.makedirs(TEST_PATH)
+
+def remove_test_directory(dir_name):
+    if os.path.isdir(dir_name):
+        shutil.rmtree(dir_name)
+
+def create_test_user(username, password, user_level):
+    test_user = User.objects.create_user(username, password)
+    test_user.user_level = user_level
+    test_user.is_active = True
+    test_user.save()
+
+def create_test_admin_user():
+    username = "test_admin"
+    password = "test_admin"
+    user_level = User.ADMIN
+    create_test_user(username, password, user_level)
+
+def create_test_judge_user():
+    username = "test_judge"
+    password = "test_judge"
+    user_level = User.JUDGE
+    create_test_user(username, password, user_level)
+
+def create_test_normal_user():
+    username = "test_user"
+    password = "test_user"
+    user_level = User.USER
+    create_test_user(username, password, user_level)
+
+def get_test_admin_user():
+    return User.objects.get(username="test_admin")
+
+def get_test_judge_user():
+    return User.objects.get(username="test_judge")
+
+def get_test_normal_user():
+    return User.objects.get(username="test_user")
+
+def get_test_admin_client():
+    test_admin_client = Client()
+    test_admin_client.login(username="test_admin", password="test_admin")
```

Comment:
This is actually a great point. Thanks for the adivice!

Thanks to django, we will not run this unit test on production DB.
Django actually provides a temporary database for testing and destroy it after testing.
Therefore, in my opinion, the implementation here is not going to be insecure.